modules/local/filter_stats/main.nf

```diff
@@ -2,10 +2,10 @@ process FILTER_STATS {
     label 'process_medium'
     tag "$meta.id"
 
-    conda 'bowtie2=2.4.5'
+    conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/bowtie2:2.4.5--py39hd2f7db1_2' :
-        'biocontainers/bowtie2:2.4.5--py39hd2f7db1_2' }"
+        'https://depot.galaxyproject.org/singularity/gawk:5.3.0' :
+        'biocontainers/gawk:5.3.0' }"
```

Comment:
This shouldn't be a problem, but for consistency, replace for the equivalent seqera container